features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/NRFCordioHCIDriver.cpp

```diff
@@ -338,20 +333,3 @@ ble::vendor::cordio::CordioHCIDriver& ble_cordio_get_hci_driver() {
 
     return hci_driver;
 }
-
-// Nordic implementation
-void PlatformLoadBdAddress(uint8_t *pDevAddr)
```

Comment:
Could you explain how the public address is loaded now ?

Reply:
PalCfgLoadData(PAL_CFG_ID_BD_ADDR, bd_addr, sizeof(bdAddr_t));